tests/test_hf_file_system.py

```diff
@@ -152,8 +152,23 @@ def test_remove_directory(self):
 
     def test_read_file(self):
         with self.hffs.open(self.hf_path + "/data/text_data.txt", "r") as f:
+            self.assertIsInstance(f, HfFileSystemFile)
```

Comment:
For me this line is failing (both locally and in the CI). Is this PR supposed to change the default behavior?

In a local python debugger, I get that `f__class__.__mro__` == `(<class '_io.TextIOWrapper'>, <class '_io._TextIOBase'>, <class '_io._IOBase'>, <class 'object'>)` (so not HfFileSystem-specific).

Reply:
Hmm this doesn't sound related to this PR but let me take a look

Reply:
Thanks!